### PR TITLE
Remove unused roles

### PR DESCRIPTION
**Pull request by fahzee1**
[TC-3592](https://whitemoondreams.atlassian.net/browse/TC-3592) - Roles: remove unused roles for now

This wont visibly be active until `PERMAFROST_DEFAULT_ROLES` is updated in TC

### CODE REVIEW

File: permafrost/views.py [hunk 1]
```diff
@@ -204,12 +211,19 @@ def get_form_kwargs(self):
...
 # List Permission Groups
 class PermafrostRoleListView(PermafrostSiteMixin, FilterByRequestSiteQuerysetMixin, ListView):
     model = PermafrostRole
     queryset = PermafrostRole.on_site.all()
     permission_required = ["permafrost.view_permafrostrole"]
+
+    def get_queryset(self):
+        qs = super(PermafrostRoleListView, self).get_queryset()
+        # Should be reflected in TC
+        return qs.filter(name__in=PERMAFROST_DEFAULT_ROLES)
```
Comment (fahzee1) [inline permafrost/views.py:224]: `PERMAFROST_DEFAULT_ROLES` will be updated to
 [
        'Student',
        'Administrator',
        'Curriculum Designer',
        'Instructor',
        'Site Owner'
]

File: permafrost/views.py [hunk 2]
```diff
@@ -252,6 +266,11 @@ def get_context_data(self, **kwargs):
         context['permissions'] = role.permissions().all().order_by("content_type").distinct()
         return context
 
+    def get_queryset(self):
+        qs = super(PermafrostRoleDetailView, self).get_queryset()
+        # Should be reflected in TC
+        return qs.filter(name__in=PERMAFROST_DEFAULT_ROLES)
+
```
Comment (fahzee1) [inline permafrost/views.py:274]: PERMAFROST_DEFAULT_ROLES will be updated to
[
'Student',
'Administrator',
'Curriculum Designer',
'Instructor',
'Site Owner'
]

File: permafrost/views.py (hunk 1 above)
Comment (mgcamisa) [reply permafrost/views.py:224]: change to exclude list instead of default list to allow users to create new roles

Comment (fahzee1) [reply permafrost/views.py:224]: good catch

**Review by mgcamisa — APPROVED** [review]

File: .idea/workspace.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ChangeListManager">
```
Comment (mgcamisa) [inline .idea/workspace.xml:3]: did you mean to include this file?

File: permafrost/views.py (hunk 2 above)
Comment (mgcamisa) [reply permafrost/views.py:274]: ^ now will be PERMAFROST_EXCLUDED_ROLES

**Review by mgcamisa — APPROVED** [review]

File: permafrost/views.py [hunk 4]
```diff
@@ -12,7 +12,14 @@
 from django.db.models import Q
 from django.views.generic.edit import CreateView
 
-from .models import PermafrostRole, get_optional_by_category, get_required_by_category, get_all_perms_for_all_categories
+from .models import (
+    PermafrostRole,
+    PERMAFROST_EXCLUDED_ROLES,
```
Comment (devo-wm) [inline permafrost/views.py:17]: shouldn't this come from settings instead of .models?

**Review by devo-wm — APPROVED** [review]

**Pull request merged**